src/state/types.ts

```diff
@@ -42,7 +42,10 @@ export type AppState = {
   deleteAccount: RemoteAction;
 
   mentors: RemoteData<Record<BuddyId, mentorsApi.Mentor>>;
-  skillFilter: string[];
+  filterMentors: {
+    skillFilter: string[];
+    hideInactiveMentors: boolean;
```

Comment:
Now that I read this again, maybe we should rename this to **shouldHideInactiveMentors** etc..